schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py

```diff
@@ -496,39 +502,29 @@ def validate_publications(publications: Optional[str]) -> Optional[str]:
 # ==============================================================================
 # Sources Validation
 # ==============================================================================
-def validate_sources(source_list: List[DefaultSource] | List[VoxelSpacingSource]) -> None:
+def validate_sources(source_list: List[StandardSource] | List[VoxelSpacingSource]) -> None:
     total_errors = []
 
-    # For verifying that all source entries each only have one finder type
-    multiple_finders_in_each_source_entries_errors = []
-    standalone_finders_in_each_source_entries_errors = []
-
-    for i, source_element in enumerate(source_list):
+    for index, source_element in enumerate(source_list):
         finders_in_source_entry = []
-        has_parent_filters = False
         for finder in source_element.model_fields:
-            # If the finder is not None, add it to the list of finders in the source entry
-            if getattr(source_element, finder) is None:
-                continue
-            if finder == "exclude":
-                continue
-            if finder == "parent_filters":
-                has_parent_filters = True
-            else:
+            # If the finder is not None and an actual finder, add it to the list of finders in the source entry
+            if getattr(source_element, finder) is not None and finder not in SOURCE_CHILDREN_NON_FINDERS:
```

Comment:
nit: Could we rename this constant `SOURCE_FILTER_FIELDS` ?

Reply:
done!